docs/htdocs/info/docs/tools/vep/script/vep_filter.html

```diff
@@ -97,6 +97,14 @@ <h2 id="filter_opt">Options</h2>
                 line to be printed
 			</td>
 		</tr>
+    <tr id="opt_softfilter">
+      <td><pre>--soft_filter</pre></td>
+      <td></td>
+      <td>
+         Variation failing the filter will not get excluded from the output file,
```

Comment:
I suggest changing to:

`Variants not passing given filters will be flagged in the FILTER column of the VCF file, and will not be removed from output.`